flipperdevices/Flipper-Android-App
# Fix file upload progress

**Background**
This PR aims to improve new file manager uploading reactivity and fix ui for multiple files

**Changes**

- Remove upload screen and replace it with component
- Add callbacks for uploaded file progreess - will now be updated in items list
- Fix multiple files upload display 

**Test plan**

- Try upload one file fully and see it will appear in list instantly
- Try upload one file but cancel it in progress. See it also appeared, but with non-full upload size
- Try upload multiple files
